requirements.txt

```diff
@@ -10,3 +10,4 @@ semver
 sentry-sdk
 thoth-common
 thoth-sourcemanagement
+thoth-glyph
```

Comment:
This should also be added to the pipfile

Reply:
Fixed